ci/aws/test-sr-iov-secondary-network-aws.sh

```diff
@@ -0,0 +1,354 @@
+        sudo apt-get update
+        sudo apt-get install containerd.io -y
+        sudo mkdir -p /etc/containerd
+        sudo containerd config default | sudo tee /etc/containerd/config.toml
+        sudo sed -i 's/SystemdCgroup = false/SystemdCgroup = true/' /etc/containerd/config.toml
+
+        sudo systemctl restart containerd
+        sudo systemctl enable containerd
+
+        echo "deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.32/deb/ /" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+        curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.32/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
+        sudo apt update
+        sudo apt install -y kubelet kubeadm kubectl
+        sudo apt-mark hold kubelet kubeadm kubectl
+        sudo swapoff -a
+        sudo sed -i '/ swap / s/^\(.*\)$/#\1/g' /etc/fstab
+EOF
+}
+
+# Function to initialize the Kubernetes master node
+initialize_master() {
+    local master_ip=$1
+    echo "Initializing Kubernetes master node on $master_ip..."
+
+    ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@"$master_ip" << EOF
+        sudo kubeadm init --pod-network-cidr=10.244.0.0/16
+        mkdir -p \$HOME/.kube
+        sudo cp -i /etc/kubernetes/admin.conf \$HOME/.kube/config
+        sudo chown \$(id -u):\$(id -g) \$HOME/.kube/config
+EOF
+}
+
+# Function to get the join command for the worker node
+get_join_command() {
+    local master_ip=$1
+    join_command=$(ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@"$master_ip" "sudo kubeadm token create --print-join-command")
+    echo "$join_command"
+}
+
+# Function to join the worker node to the cluster
+join_worker() {
+    local worker_ip=$1
+    local join_command=$2
+    echo "Joining worker node at $worker_ip to the Kubernetes cluster..."
+
+    ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@$worker_ip << EOF
+        sudo $join_command
+EOF
+}
+
+# Function to verify the Kubernetes cluster status
+verify_cluster() {
+    local master_ip=$1
+    echo "Verifying Kubernetes cluster status on master node $master_ip..."
+
+    ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@$master_ip << EOF
+        kubectl get nodes
+EOF
+}
+
+function setup_cluster {
+    # Launch Master and Worker EC2 instances
+    echo "Launching EC2 instance ..."
+    MASTER_INSTANCE_ID=$(launch_ec2_instance "ControlPlane")
+    echo "ControlPlane EC2 instance launched with Instance ID: $MASTER_INSTANCE_ID"
+    WORKER_INSTANCE_ID=$(launch_ec2_instance "Worker")
+    echo "Worker EC2 instance launched with Instance ID: $WORKER_INSTANCE_ID"
+
+    attach_network_interface "$MASTER_INSTANCE_ID"
+    attach_network_interface "$WORKER_INSTANCE_ID"
+
+    echo "========MASTER_INSTANCE_ID: $MASTER_INSTANCE_ID, MASTER_INSTANCE_ID: $WORKER_INSTANCE_ID========="
```

Comment:
Leave a space between words and '===', or remove "========" directly.
```suggestion
    echo "======== MASTER_INSTANCE_ID: $MASTER_INSTANCE_ID, WORKER_INSTANCE_ID: $WORKER_INSTANCE_ID ========="
```